crypto/merklearray/merkle.go

```diff
@@ -31,15 +31,15 @@ type Tree struct {
 	_struct struct{} `codec:",omitempty,omitemptyarray"`
 
 	// Level 0 is the leaves.
-	Levels []Layer            `codec:"lvls,allocbound=-"`
+	Levels []Layer            `codec:"lvls,allocbound=9999"`
```

Comment:
Please add a constant for this one. The actual number should be much smaller than 9999. Most likely 256 is a good limit.